### PR TITLE
fix: use default export for webpack5 wrapper(#619)

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

* related: #619 
* webpack issue: https://github.com/webpack/webpack/issues/12995

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/react-chart/src/base.js [hunk 1]
```diff
@@ -65,7 +29,7 @@ export default function (chartType) {
 
     componentDidMount() {
       const { data, options } = this.props;
-      this.chartInst = new creator[chartType]({ el: this.rootEl.current, data, options });
+      this.chartInst = Chart[`${chartType}Chart`]({ el: this.rootEl.current, data, options });
```
Comment (js87zz) [inline apps/react-chart/src/base.js:32]: 다시봐도 왜 인식을 못하는지 참..이상하네요.. 버그가 얼른해결되면 좋겠군요.

**Review by js87zz — APPROVED** [review]
리뷰 완료입니다!

**Review by jungeun-cho — APPROVED** [review]
리뷰완료합니다.
해당 브랜치 머지된 후,
새로 추가된 gauge차트 부분 래퍼쪽 추가작업을 이어서 다른 브랜치에서 작업할 예정입니다. :)

**Review by lja1018 — APPROVED** [review]
리뷰완료
고생하셨습니다!

**Review by shiren — APPROVED** [review]
수고하셨습니다

**Review by dotaitch — APPROVED** [review]
리뷰 완료 합니다

**Pull request merged**